[2024-12-19T07:18:55.389+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-19T07:18:55.401+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v2.second_task scheduled__2022-02-18T00:00:00+00:00 [queued]>
[2024-12-19T07:18:55.413+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v2.second_task scheduled__2022-02-18T00:00:00+00:00 [queued]>
[2024-12-19T07:18:55.414+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-19T07:18:55.430+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): second_task> on 2022-02-18 00:00:00+00:00
[2024-12-19T07:18:55.439+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=10774) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-19T07:18:55.441+0000] {standard_task_runner.py:72} INFO - Started process 10793 to run task
[2024-12-19T07:18:55.441+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'second_task', 'scheduled__2022-02-18T00:00:00+00:00', '--job-id', '2089', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpl4d_304c']
[2024-12-19T07:18:55.449+0000] {standard_task_runner.py:105} INFO - Job 2089: Subtask second_task
[2024-12-19T07:18:55.497+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v2.second_task scheduled__2022-02-18T00:00:00+00:00 [running]> on host 13c961a72918
[2024-12-19T07:18:55.568+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='code2j' AIRFLOW_CTX_DAG_ID='our_first_dag_v2' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2022-02-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-02-18T00:00:00+00:00'
[2024-12-19T07:18:55.570+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-19T07:18:55.583+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-12-19T07:18:55.584+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and will be running after task1']
[2024-12-19T07:18:55.602+0000] {subprocess.py:99} INFO - Output:
[2024-12-19T07:18:55.606+0000] {subprocess.py:106} INFO - hey, I am task2 and will be running after task1
[2024-12-19T07:18:55.609+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2024-12-19T07:18:55.639+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-19T07:18:55.640+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_task, run_id=scheduled__2022-02-18T00:00:00+00:00, execution_date=20220218T000000, start_date=20241219T071855, end_date=20241219T071855
[2024-12-19T07:18:55.672+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-19T07:18:55.680+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
